propolis/src/hw/nvme/mod.rs

```diff
@@ -1005,7 +1011,7 @@ lazy_static! {
         // Pad out to the next power of two
```

Comment:
This comment should probably be updated to reflect the new nature of the padding?

Reply:
Updated